write-fonts/src/tables/variations.rs

```diff
@@ -21,6 +21,237 @@ impl VariationRegionList {
     }
 }
 
+/// <https://learn.microsoft.com/en-us/typography/opentype/spec/otvarcommonformats#packed-point-numbers>
+pub struct PackedPointNumbers {
+    is_all: bool,
```

Comment:
smells like an enum?

Reply:
hahaha I turn this into an enum in a commit on #319 💁